services/search.py

```diff
@@ -16,18 +16,23 @@ def get(self):
 
         s = urllib.unquote_plus(s)
 
+        query = '"{0}"'.format(s.replace('"', ''))
         stream_list = []
 
         expr_list = [search.SortExpression(
             expression='name', default_value='',
-            direction=search.SortExpression.DESCENDING)]
+            direction=search.SortExpression.DESCENDING), 
+            search.SortExpression(
```

Comment:
I also added this one, I think I was finding tags with this, but not sure anymore. Feel free to remove it.